tyk-single-dc/README.md

```diff
@@ -0,0 +1,208 @@
+## Tyk Tyk single dc
+
+`tyk-single-dc` provides the default deployment of a Tyk Single Data Center. It will deploy the data center components with the settings provided in the values.yaml file.
+
+It includes:
+- the Tyk Gateway, an open source Enterprise API Gateway (supporting REST, GraphQL, TCP and gRPC protocols)
+- Tyk Pump, an analytics purger that moves the data generated by your Tyk nodes to any back-end. Furthermore, it has all the required modifications to easily connect to Tyk Cloud or Multi Data Center (MDCB) control plane.
+- Tyk Dashboard, a license based component that allows you to interact with Tyk in a browser
+
+## Introduction
+
+By default, this chart installs following components as subcharts on a [Kubernetes](https://kubernetes.io/) cluster using the [Helm](https://helm.sh/) package manager.
+| Component | Enabled by Default | Flag |
+| --------- | ------------------ | ---- |
+|Tyk Gateway |true  | n/a                    |
+|Tyk Pump    |true | global.components.pump |
+|Tyk Dashboard| true| global.components.dashboard | 
```

Comment:
@andrei-tyk is tyk-dashboard an optional component? Can I turn it off to install additional gateway that connect to the dashboard?

Reply:
yes and yes :D